src/config.rs

```diff
@@ -96,6 +96,7 @@ pub struct DatasetConfigModel {
     pub aliases: Vec<String>,
     pub sqd_network: Option<DatasetRef>,
     pub real_time: Option<RealTimeConfig>,
+    pub kind: String,
```

Comment:
It should be optional in the config. Most users will not provide it and fall back to the kind set in the mapping file